src/main/scala/com/stratio/hermes/dao/ZookeeperConfigDAO.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2016 Stratio (http://stratio.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.stratio.hermes.dao
+
+import com.stratio.hermes.constants.HermesConstants
+import com.stratio.hermes.exceptions.HermesException
+import org.apache.curator.framework.{CuratorFramework, CuratorFrameworkFactory}
+import org.apache.curator.retry.ExponentialBackoffRetry
+
+import scala.util.{Failure, Success, Try}
+
+class ZookeeperConfigDAO extends ConfigDAO[String] {
+  val config = com.stratio.hermes.implicits.HermesImplicits.config
+  lazy val curatorFramework: CuratorFramework = buildCurator
+  lazy val connectionString = Try(config.getString(HermesConstants.ZookeeperConnection)).getOrElse(HermesConstants.ZookeeperConnectionDefault)
+  lazy val connectionTimeout = config.getInt(HermesConstants.ZookeeperConnectionTimeout)
+  lazy val sessionTimeout = config.getInt(HermesConstants.ZookeeperSessionTimeout)
+  lazy val retryAttempts = config.getInt(HermesConstants.ZookeeperRetryAttempts)
+  lazy val retryInterval = config.getInt(HermesConstants.ZookeeperRetryInterval)
+  lazy val parentNodePath = config.getString(HermesConstants.ZookeeperParentNodePath)
+
+  override def saveConfig(path: String, config: String): Unit = Try(
+    if (existsConfig(path)) {
+      updateConfig(path, config)
+    }
+    else {
+      curatorFramework.create().creatingParentsIfNeeded().forPath("/" + parentNodePath + path)
```

Comment:
Why are you putting "/" in all places? I think that it could be better to have parentNode = "/stratio/hermes" instead of "stratio/hermes". Furthermore it is something "static", in my opinion is better to have it the constants instead of a configuration file that possibly never will be change. Also I think that it is more legible to use string interpolation to build strings: s"$parentNodePath/$path"